### PR TITLE
feat: send error log

### PR DESCRIPTION
**Pull request by pchila**
Send uniform error log for job-executor-service errors occurring before any job can be started.
Remove AlwaysSendFinishedEvent flag from JobSetting structure

Screenshots:

Screenshot 2022-04-28 at 15-18-18 keptn
Screenshot 2022-04-28 at 15-18-38 keptn

### CODE REVIEW

File: pkg/eventhandler/eventhandlers.go [hunk 1]
```diff
@@ -62,6 +67,7 @@ type EventHandler struct {
 	ImageFilter     ImageFilter
 	Mapper          EventMapper
 	K8s             K8s
+	ErrorSender     UniformErrorLogSender // TODO initialize correctly
```
Comment (christian-kreuzberger-dtx) [inline pkg/eventhandler/eventhandlers.go:70]: what abou tthe Todo?

Comment (pchila) [reply pkg/eventhandler/eventhandlers.go:70]: It was done but I forgot to remove it :stuck_out_tongue_winking_eye:

File: pkg/keptn/errorlog.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+	if initialCloudEvent == nil {
+		return ErrorInitialCloudEventNotSpecified
+	}
+
+	if applicationError == nil {
+		return ErrorProcessingErrorNotSpecified
+	}
+
+	// uniformHandler := api.NewUniformHandler("localhost:8081")
```
Comment (Raffy23) [inline pkg/keptn/errorlog.go:71]: I think we can remove this comment, as the uniform handler is passed as parameter

**Review by Raffy23 — DISMISSED** [review]
LGTM

**Review by christian-kreuzberger-dtx — APPROVED** [review]
also LGTM from my side

**Review by Raffy23 — APPROVED** [review]

**Pull request merged**